specification/healthbot/resource-manager/Microsoft.HealthBot/stable/2020-10-20/healthbot.json

```diff
@@ -0,0 +1,763 @@
+      "type": "object",
+      "properties": {
+        "nameAvailable": {
+          "description": "indicates if the bot name is valid and available.",
+          "type": "boolean"
+        },
+        "reason": {
+          "description": "Reason for the result",
+          "type": "string"
+        },
+        "message": {
+          "description": "additional message from server",
+          "type": "string"
+        }
+      }
+    },
+    "Resource": {
+      "description": "The resource model definition for a ARM tracked top level resource",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "description": "Fully qualified resource Id for the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "The name of the resource",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The type of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "systemData": {
+          "$ref": "#/definitions/SystemData",
+          "readOnly": true,
+          "description": "Metadata pertaining to creation and last modification of the resource"
+        }
+      }
+    },
+    "TrackedResource": {
+      "description": "The resource model definition for a ARM tracked top level resource",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "x-ms-mutability": [
+            "read",
+            "create",
+            "update"
+          ],
+          "description": "Resource tags."
+        },
+        "location": {
+          "type": "string",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ],
+          "description": "The geo-location where the resource lives"
+        }
+      },
+      "required": [
+        "location"
+      ],
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ]
+    },
+    "HealthBot": {
+      "type": "object",
+      "description": "HealthBot resource definition",
+      "allOf": [
+        {
+          "$ref": "#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/HealthBotProperties",
+          "description": "The set of properties specific to healthcare bot resource."
+        }
+      }
+    },
+    "HealthBotProperties": {
+      "title": "HealthBotProperties",
+      "description": "The properties provided for bot creation. The Health Bot Service is a cloud platform that empowers developers in Healthcare organizations to build and deploy their compliant, AI-powered virtual health assistants and health bots, that help them improve processes and reduce costs.",
```

Comment:
How does a user get this value from the bot after its created?  There are no APIs exposed here other than general crud on the bot

Reply:
Basically, once an end-user creates a HealthBot, we generate a link to the bot management portal, that is accessible to the identity (mail) of the resource creator. Should the link to the management portal (and the mail of the owner) be part of the HealthBot properties?

Reply:
@pilor  Basically, once an end-user creates a HealthBot, we generate a link to the bot management portal, that is accessible to the identity (mail) of the resource creator (or to the owners of the Azure resource, we are still discussing it internally).
We initially thought that we would generate and show the link in the Azure Portal, but I totally understand your point - since the end user may use an ARM template to create a HealthBot, I understand that it should be in the HealthBot properties.

Reply:
I added a new property called "botManagementPortalLink". Please let me know if this is what you meant.